python/faiss.py

```diff
@@ -35,15 +39,15 @@ def instruction_set():
 try:
     instr_set = instruction_set()
     if instr_set == "AVX2":
-        print("Loading faiss with AVX2 support.", file=sys.stderr)
+        logger.info("Loading faiss with AVX2 support.")
         from .swigfaiss_avx2 import *
     else:
-        print("Loading faiss.", file=sys.stderr)
+        logger.info("Loading faiss.")
         from .swigfaiss import *
 
 except ImportError:
     # we import * so that the symbol X can be accessed as faiss.X
-    print("Loading faiss.", file=sys.stderr)
+    logger.error("Loading faiss.")
```

Comment:
Does this need to be on error level? Afaik this is expected to happen if the platform support avx2 but faiss is compiled without `swigfaiss_avx2`.

Reply:
All of the logging could be info or lower level. What is the expectation of swigfaiss_avx2 build for general use? I couldn't find documentation on avx2 support in `INSTALL.md`